src/test/kotlin/com/personio/synthetics/builder/api/StepBuilderTest.kt

```diff
@@ -146,6 +150,43 @@ class StepBuilderTest {
         assertEquals(isCritical, result.isCritical)
     }
 
+    @Test
+    fun `build sets step retry defaults if retry method was not called`() {
```

Comment:
A bit unclear phrasing, did you mean: builder sets the default retry count and interval if the retry method is not called?